### PR TITLE
Adding modelx package

### PR DESCRIPTION
**Pull request by fumitoh**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by fumitoh** [comment]
@conda-forge-admin ready for review

File: recipes/modelx/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Copyright (c) 2017-2022 Fumito Hamamura <fumito.ham@gmail.com>
```
Comment (beenje) [inline recipes/modelx/LICENSE.txt:1]: Both LICENSE.txt and LICENSE.lesser.txt are included in the pypi package. No need to copy that file here.

File: recipes/modelx/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/fumitoh/modelx
+  summary: Numerical computing tool to use Python like a spreadsheet
+  license: LGPL-3.0-only
+  license_file: LICENSE.txt
```
Comment (beenje) [inline recipes/modelx/meta.yaml:38]: Use LICENSE.LESSER.txt which is included in the package (it's the one corresponding to LGPL).

File: recipes/modelx/LICENSE.txt (hunk 1 above)
Comment (fumitoh) [reply recipes/modelx/LICENSE.txt]: baee2f14f9ba8c04ea2c3bd6022ee97c00b09244 removes the file.

File: recipes/modelx/meta.yaml (hunk 2 above)
Comment (fumitoh) [reply recipes/modelx/meta.yaml]: baee2f14f9ba8c04ea2c3bd6022ee97c00b09244 fixes the file name.

**Comment by beenje** [comment]
@conda-forge/help-python, this recipe looks good to me

**Pull request merged**